js/controllers/hotkeys.js

```diff
@@ -1,3 +1,26 @@
+/**
+ * Enum for keyboard event keycodes
+ * @enum {number}
+ */
+const Keycode = {
+  TAB: 9,
+  ENTER: 13,
+  ESCAPE: 27,
+  SPACE: 32,
+  ZERO: 48,
+  NUMPAD_ZERO: 96,
+  NUMPAD_PLUS: 107,
+  NUMPAD_MINUS: 109,
+  PLUS: 187,
+  MINUS: 189
```

Comment:
Note for future: add a trailing comma to the end of object literals like this :)

Reply:
Ack.